src/features/Home/TipsModal/index.tsx

```
@@ -77,69 +77,92 @@ const TipsModal = ({
+                )}
               </div>
               <div className={s.TipsModal_Todo}>
                 <ul className={s.Todo_Content}>
-                  {Tips?.todos?.map((item: TodoProps) => {
-                    return <li className={s.Todo_Title}>{item.title}</li>;
-                  })}
+                  {Tips.todos.length > 0 ? (
+                    Tips?.todos?.map((item: TodoProps) => {
+                      return <li className={s.Todo_Title}>{item.title}</li>;
+                    })
+                  ) : (
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)